code/modules/mapping/mapping_helpers.dm

```diff
@@ -27,6 +27,8 @@
 			formatted[i] = TRUE
 		baseturf_to_replace = formatted
 
+	baseturf_to_replace += list(/turf/baseturf_bottom = TRUE)
```

Comment:
This should get added in manually not automatically be there every time

Reply:
/turf/baseturf_bottom the placeholder thet creates Z lvl baseturf when placed.
you really want to say every mapper that say "baseturf helper dont work" what it want to add /turf/baseturf_bottom in list of baseturfs to replace?

Ok, i just add /turf/baseturf_bottom to the initial baseturf_to_replace,
as i think we dont have old manual baseturfs, only /turf/baseturf_bottom and baseturf_helper.

Reply:
If you think this can be used to just add some baseturfs, baseturf_bottom need  to be lost bottom, otherwise it blocks work of baseturf_helper

Reply:
i think this wrong, but i do this